frontend/webpack.conf.js

```diff
@@ -34,16 +38,18 @@ module.exports = function(debug) { return {
         ]
     },
 
+    resolveLoader: {
+        modules: [path.join(__dirname, "node_modules")]
+    },
+
     plugins: !debug ? [
-        new Uglify({compress: {warnings: false}})
+        new Uglify(
+            {
+                sourceMap: true
+            })
```

Comment:
From: https://webpack.js.org/guides/migrating/#uglifyjsplugin-sourcemap

> The `sourceMap` option of the UglifyJsPlugin now defaults to `false `instead of `true`.